packages.txt

```diff
@@ -0,0 +1 @@
+pandoc
```

Comment:
Where is `pandoc` used? I do not remember... 🤔

Reply:
This is generated by the template file "tmpl/_building/tmpl/requirements.txt"
 I don't know what the purpose of this file is😗

Reply:
Yeah it's a template that has some useless demo things there.
Forget what you are taught, trust your judgement.

Reply:
> Yeah it's a template that has some useless demo things there.
> Forget what you are taught, trust your judgement.
i got it😁